processor/k8sattributesprocessor/README.md

```diff
@@ -271,10 +271,12 @@ k8sattributes/2:
       - k8s.node.name
       - k8s.pod.start_time
     labels:
-     # This label extraction rule takes the value 'app.kubernetes.io/component' label and maps it to the 'app.label.component' attribute which will be added to the associated resources
-     - tag_name: app.label.component
-       key: app.kubernetes.io/component
-       from: pod
+      # This label extraction rule takes the value 'app.kubernetes.io/component' label and maps it to the 'app.label.component' attribute which will be added to the associated resources
+      - tag_name: app.label.component
+        key: app.kubernetes.io/component
+        from: pod
+    # Apply the operator rules - see https://github.com/open-telemetry/semantic-conventions/blob/main/docs/non-normative/k8s-attributes.md
+    otel_annotations: true # default is false - use true to enable
```

Comment:
I'm fine with the addition of this specific one. However, I wonder how this will be combined with the rest of the settings as described at https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37114#discussion_r2035380275. Sorry if I miss any context here but that'd help clarifying this.

Reply:
I think we'll need to have another 2 flag to cover this

Reply:
Should we decide on those at a high level before adding this specific one? My preference would be to have alignment on the high level config schema before starting any partial implementations. Would that make sense?

Reply:
I don't think we need any other config options for other things. See https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37114/files#r2025819122

Reply:
What about the well known labels that are described at https://github.com/open-telemetry/opentelemetry-operator?tab=readme-ov-file#configure-resource-attributes-with-labels and was originally added in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37114/files#diff-57b0d6448d516c0c80b326a7f7f95c3256b2b774e496870dc1dd35dcf8bfc256R281?

Reply:
Hmm, `extract::metadata` is a map so I'm not sure how we could add extra options without breaking it, right?
Also in this PR we extract the annotation through the `extract::otel_annotations` setting so I assume the equivalent would be `extact::well_known_labels`, @zeitlinger could you confirm if that's the direction you had in mind?

Reply:
From my perspective I'm coverd with the current state and the future plans. @dmitryax feel free to resolve this comment if we are on agreement.